nibabel/tests/test_loadsave.py

```diff
@@ -135,3 +136,33 @@ def test_read_img_data_nifti():
             assert_array_equal(exp_offset, read_img_data(img_back))
             # Delete stuff that might hold onto file references
             del img, img_back, data_back
+
+
+def test_save():
+    with InTemporaryDirectory():
+        dataobj = np.ones((10, 10, 10), dtype=np.float16)
+        affine = np.eye(4, dtype=np.float32)
+        img = SpatialImage(dataobj, affine)
+
+        # The `save` method can raise one of many types of errors, but they are
+        # all subclasses of Exception. This attempt will fail because float16
+        # is not supported.
+        with assert_raises(Exception):
+            save(img, 'foo.nii.gz')
+
+        # Test the saving of several types of images.
```

Comment:
I believe we have tests for these elsewhere - would you mind having a quick look?  Otherwise I will.

Reply:
I should have expected there to be tests for this. I think they are in [nibabel.tests.test_image_load_save](https://github.com/nipy/nibabel/blob/master/nibabel/tests/test_image_load_save.py). I moved the test to that file and removed the redundant tests.